index.html

```diff
@@ -165,15 +185,17 @@ <h3 class="languages-title">Languages</h3>
         </div>
     </section>
     <div class= "contact-me">
-        <h2 class="Contact-me-title">Contact me</h2>
-        <p class="Contact-me-body">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
-        <form class="Contact-form" action="https://formspree.io/f/mgebklpe
-        " method="post">
-            <input type="text" id="name" name="name" placeholder="Write your name here!" required maxlength="30" />
-            <input type="email" id="email" name="email" placeholder="Write your e-mail here!" required />
-            <textarea name="user-massage" id="textarea" cols="30" rows="10" placeholder="Write something here" required maxlength="500"></textarea>
-            <button type="submit" class="btn normal-b" id="contact-me-action"> Get in touch </button>
-        </form>
+        <div class="contact-me-div">
+            <h2 class="Contact-me-title">Contact me</h2>
+            <p class="Contact-me-body">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
+            <form class="Contact-form" action="https://formspree.io/f/mgebklpe
+            " method="post">
+                <input type="text" id="name" name="name" placeholder="Write your name here!" required maxlength="30" />
+                <input type="email" id="email" name="email" placeholder="Write your e-mail here!" required />
+                <textarea name="user-massage" id="textarea" cols="30" rows="10" placeholder="Write something here" required maxlength="500"></textarea>
+                <button type="submit" class="btn normal-b" id="contact-me-action"> Get in touch </button>
+            </form>
+        </div>
```

Comment:
Hey 👋🏽 there's some overflow issue here on the mobile version. Take a look 👇🏽 

![Screenshot 2023-02-24 at 08 37 13](https://user-images.githubusercontent.com/45739963/221192402-387efa99-2a6e-4b8a-b96e-96eace33f340.png)

Try to address that as the design should be working on both web and mobile. Best 🎉